packages/editor/src/utils/server-side-render/index.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * WordPress dependencies.
+ */
+import { ServerSideRender } from '@wordpress/components';
+import { withSelect } from '@wordpress/data';
+
+export default withSelect( ( select ) => {
+	const { getCurrentPostId } = select( 'core/editor' );
+
+	return {
+		urlQueryArgs: {
```

Comment:
We are ignoring the urlQueryArgs we may be receiving. Maybe we can just check if in urlQueryArgs props this component receives the property post_id is not set and if that's case we add it. urlQueryArgs would default to "{}" so on the most common case we would only add the post_id (equivalent to what we have now).

Reply:
Good catch! I've updated the implementation for this. It no longer uses `withSelect` but just returns a functional component instead.